packages/vue/src/components/atoms/SfChevron/SfChevron.vue

```diff
@@ -1,9 +1,28 @@
-<template>
-  <div class="sf-chevron">
+<template functional>
+  <div
+    :class="[data.class, data.staticClass, 'sf-chevron']"
+    :style="[data.style, data.staticStyle]"
+    v-bind="data.attrs"
+    v-on="listeners"
+  >
     <!-- @slot Custom chevron markup -->
     <slot>
-      <span class="sf-chevron__bar sf-chevron__bar--left"></span>
-      <span class="sf-chevron__bar sf-chevron__bar--right"></span>
+      <span
+        :class="[
+          data.class,
```

Comment:
If you add like this, it means the children will have the same class with the wrappers? For example if I use
<sf-chevron class="myclass"></sf-chevron>
it will render
<div class="myclass sf-chevron">
<span class="myclass sf-chevron__bar sf-chevron__bar--left"></span>
<span class="myclass sf-chevron__bar sf-chevron__bar--right"></span>
</div>

I don't think this is the correct behavior. Plus I think by default classes are already inherited to the main wrapper. Why do we need to specify?

Reply:
You are right it should be there I remove them. Thank you :)